dev-python/awkward/awkward-2.5.2.ebuild

```diff
@@ -0,0 +1,27 @@
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+DISTUTILS_USE_PEP517=hatchling
+export SETUPTOOLS_SCM_PRETEND_VERSION=${PV}
+inherit distutils-r1 pypi
+
+DESCRIPTION="Manipulate JSON-like data with NumPy-like idioms."
+HOMEPAGE="https://github.com/scikit-hep/awkward"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	>=dev-python/numpy-1.18.0[${PYTHON_USEDEP}]
+	~dev-python/awkward-cpp-28[${PYTHON_USEDEP}]
+	dev-python/packaging[${PYTHON_USEDEP}]
+"
+BDEPEND="
+	${RDEPEND}
```

Comment:
Probably not all the RDEPENDs are BDEPENDs, packaging sounds like it might be BDEPEND.